### PR TITLE
Fixed bonfire selected tab bug

### LINKED ISSUE #196: Selecting text and tab indenting it
I'm not sure how complicated this would be to implement, but it would be nice if users could select text in bonfire and hit tab to tab in all selected text. The current behavior is the tab overwrites all selected text.

### PR DESCRIPTION
**Pull request by MrRenter**
If you selected text in a bonfire and pressed tab it would delete the text instead of tabbing.
Closes https://github.com/FreeCodeCamp/freecodecamp/issues/196

### CODE REVIEW

File: controllers/home.js [hunk 1]
```diff
@@ -4,11 +4,11 @@
  */
 
 exports.index = function(req, res) {
-  if (req.user) {
-    res.redirect('/learn-to-code')
-  } else {
+if (reg.user && process.env.NODE_ENV !== 'production'){ //added !== dev because crash
+	res.redirect('/learn-to-code')
```
Comment (BerkeleyTrue) [inline controllers/home.js]: Can you clarify why you are checking if we are not in production here?

File: controllers/home.js [hunk 2]
```diff
@@ -4,11 +4,11 @@
...
+} else {
     res.render('home', {
       title: 'Learn to Code and Become a Software Engineer'
     });
-  }
+}
 };
```
Comment (BerkeleyTrue) [inline controllers/home.js]: Please fix indenting on if else.

File: controllers/home.js (hunk 1 above)
Comment (MrRenter) [inline controllers/home.js]: If you clone the repo and try to run the site if you attempt to access challenges or bonefire it crashes the server. I discussed with nathan and this was the agreed solution.

Comment (BerkeleyTrue) [inline controllers/home.js]: I am seeing 'Failed to lookup view "resources/learn-to-code"' is this what you are seeing?

**Comment by QuincyLarson** [comment]
This does fix the indention while selected. I've applied it to both the Bonfires and the Coursewares. Because there is some code on this pull request that we definitely don't want in master, I've opted to just make the changes myself rather than wait for @MrRenter to submit a subsequent pull request.

Thanks you for your help, @MrRenter.

**Pull request closed without merging**